tools/dind/sync.js

```diff
@@ -10,8 +10,8 @@ Usage:
 The script will first build a non-optimized version of the js code and sync the resulting app.js
```

Comment:
I'm not sure why you moved this file to `tools/dind`, it's not dind specific in any way (and from memory not normally used for the dind supervisor at all)

Reply:
oh it's meant to be tools/ only, that's weird